tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
That's a part of `fishline` that is currently broken / hard to explain. Its just a custom script that shows you all segment in their context. It broke in `fish 2.6` due to a bug (which is fixed on master) (see https://github.com/fish-shell/fish-shell/issues/4206). The thing is you should be able to test a segment by running `test/run.fish screen` and it will run this script in an enclosed fish session with `fishline` sourced from the git repository you are running `test/run.fish` from.